packages/features/sales-escrows/modals/StartEscrowModal.tsx

```diff
@@ -17,6 +17,7 @@ export type StartEscrowModalProps = {
   open: boolean;
   handleClose: any;
   onSuccess: any;
+  setInProcess: (any) => void;
```

Comment:
1. Any should be boolean
2. Please rename to onProcessing
3. Please make it optional